### ISSUE TITLE
Need.Video recording function available

### ISSUE THREAD

**Issue by paundri** [mgks/Android-SmartWebView#212]
How to make video recording button enable, when i open file input from camera, it is open camera, but cannot switch into vide recording mode...

**Comment by mgks** [comment]
You need to check the input file type, whether it is set to `video/*`.

**Comment by paundri** [comment]
Yes indeed, file input is set to video/*, but still cannot switch from camera mode to video or video button is disable mode.

**Comment by mgks** [comment]
try setting `ASWV_F_TYPE` to `*/*` for all file types and make sure you're requesting `video/*` in your HTML form.

**Comment by mgks** [comment]
Fixed | Working | Duplicate #323
